### ISSUE TITLE
JSON serialization does not escape special character

### ISSUE THREAD

**Issue by maxleiko** [dukeboard/kevoree-modeling-framework#77]
If your model contains:

```
class MyClass {
  name: String
}
```

Then you can do something like this:

```java
MyClass c = view.createMyClass().setName("foo\nbar");
System.out.println(c);
```

Which prints:

```
{
	"@meta" : "MyClass",
	"@uuid" : "1",
	"name":"foo
bar",
}
```

This JSON output is not valid.

**Comment by brice-morin** [comment]
Could be related to #11 and #18 

This should be better tested to avoid regression.

**Comment by gnain** [comment]
Thanks for the history @brice-morin. However, Issues #11 and #18 were related to KMF version 3, which has somehow nothing to do with the current version.
If you wanna help us by adding meaningful unit tests, you're more than welcome.

**Comment by dukeboard** [comment]
Fixed in 4.9.1, thanks maxime, tell me if you still face some issues on JSON format
